README.md

```diff
@@ -136,25 +185,25 @@ This environment variable is used to set the database password for the app datab
 
 Example values: `78sb6g654b56sdv7s89dv` or `as78sdv78sfdv67sdv5dc8sdv09sv`
 
-#### `JwtBearerSecretKey`
+#### `Authentication:JwtBearer:SecretKey`
 
 This environment variable is used to set the key used for signing the JWT token used for user authorization. NOTE: this must be a string that is at least 128 bit.
 
 Example values: `YzhSGXWIWqjDvSlUbDUWC6jlPGy9vdlPPHnDHI63v50wcN4l1k4BreFaw6pC5BgUlBzyxvhBhpokNrg9lfmjt2V7geo9CniLV2bu` or `pl0vyh6y3tDvHHCBcfj6knlhN9lW9FjDGUGxgLNeqz2XbU7rpdVleDIWuSa2hmYvwBglir4Bvzk5VogbPwl46U5InSWPP1BCo9jC`
 
-#### `EMAIL_USER`
+#### `Email:User`
```

Comment:
This environment variable should still be EMAIL_USER, just like the other ones should.
These are the variables used to replace values in the docker-compose.yml inside ${...} and not necessarily the same as the ones the container expects.

Please change them to be in line with the names in the docker-compose.yml, since they have the preferred format (capitalised snake-case)

Please also check the other environment variables.